package.json

```diff
@@ -49,10 +49,10 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@capacitor/android": "^7.0.0",
-    "@capacitor/core": "^7.0.0",
+    "@capacitor/android": "^8.0.0-beta.0",
```

Comment:
Update @capacitor/cli, @capacitor/core, @capacitor/android and @capacitor/iOS in this file to use next tag.